etc/Makefile

```diff
@@ -156,6 +156,9 @@ ifdef MULTIARCH_BUILD
 	(cd ../build/docker/$1; docker buildx build ${NO_CACHE} --platform linux/arm64,linux/amd64 --build-arg HUB_ORG=${HUB_ORG} --build-arg TAG=${TAG} --build-arg SPARK_VERSION=${SPARK_VERSION} -t $(HUB_ORG)/$1:$(TAG) . --push)
 	@echo "remove builder instance $1"
 	-(docker buildx rm $1)
+else ifeq ($(TARGET_ARCH), $(filter $(TARGET_ARCH), linux/arm64 linux/amd64))
```

Comment:
Hmm.  I'm seeing an empty `--platform` entry when no target arch is specified.  (I believe CI is behaving the same as well.):
```bash
$ make kernel-py
make WHEEL_FILE=dist/jupyter_enterprise_gateway-3.2.0.dev0-py3-none-any.whl VERSION=3.2.0.dev0 NO_CACHE= TAG=dev SPARK_VERSION=3.2.1 MULTIARCH_BUILD= TARGET_ARCH= -C etc kernel-py
Untagged: elyra/kernel-py:dev
rm -f ../dist/jupyter_enterprise_gateway_kernel_image_files-3.2.0.dev0.tar.gz
a .
a ./bootstrap-kernel.sh
a ./kernel-launchers
a ./kernel-launchers/python
a ./kernel-launchers/R
a ./kernel-launchers/scala
a ./kernel-launchers/scala/lib
a ./kernel-launchers/scala/lib/toree-launcher_2.12-3.2.0.SNAPSHOT.jar
a ./kernel-launchers/scala/lib/toree-assembly-0.5.0-incubating.jar
a ./kernel-launchers/R/scripts
a ./kernel-launchers/R/scripts/server_listener.py
a ./kernel-launchers/R/scripts/launch_IRkernel.R
a ./kernel-launchers/python/scripts
a ./kernel-launchers/python/scripts/launch_ipykernel.py
(cd ../build/docker/kernel-py; docker build  --platform  --build-arg HUB_ORG=elyra --build-arg TAG=dev --build-arg SPARK_VERSION=3.2.1 -t elyra/kernel-py:dev .)
"docker build" requires exactly 1 argument.
See 'docker build --help'.

Usage:  docker build [OPTIONS] PATH | URL | -

Build an image from a Dockerfile
make[1]: *** [../.image-kernel-py] Error 1
gmake: *** [Makefile:160: kernel-py] Error 2
```